getting-started-guides/csp/gcp/gcp.md

```diff
@@ -28,52 +27,56 @@ There are three steps to run a sample PySpark XGBoost app using Jupyter notebook
 
 ### Step 1.  Initialization steps to download required files for Spark RAPIDS XGBoost PySpark app
 
-Before you create a cluster, please git clone the [spark-examples directory](https://github.com/rapidsai/spark-examples) to your local machine. `cd` into the spark-examples/getting-started-guides/csp/gcp/spark-gpu directory. Open the rapids.sh script using a text editor.  Modify the `STORAGE_BUCKET=my-bucket` line to specify your google GCP bucket name.  
+Before you create a cluster, please git clone the [spark-examples directory](https://github.com/rapidsai/spark-examples) to your local machine. `cd` into the spark-examples/getting-started-guides/csp/gcp/spark-gpu directory. Open the rapids.sh script using a text editor.  Modify the `GCS_BUCKET=my-bucket` line and `INIT_ACTIONS_BUCKET=my-bucket`(this parameter will not be required after init action merged into dataproc official repo) to specify your google GCP bucket name.  
 
 Execute the commands below while in the spark-examples folder.  These commands will copy the following files into your GCP bucket: 
 
 1. Initialization scripts for GPU and RAPIDS Spark, 
 2. PySpark app files  
 3. A sample dataset for a XGBoost PySpark app 
-4.  The latest Spark RAPIDS XGBoost jar files from the public maven repository
+4. The latest Spark RAPIDS XGBoost jar files from the public maven repository
 
 ```
-export STORAGE_BUCKET=my-bucket
+export GCS_BUCKET=my-bucket
 pushd getting-started-guides/csp/gcp/spark-gpu
-gsutil cp -r internal gs://$STORAGE_BUCKET/spark-gpu/
-gsutil cp rapids.sh gs://$STORAGE_BUCKET/spark-gpu/rapids.sh
+gsutil cp -r internal gs://$GCS_BUCKET/spark-gpu/
+gsutil cp rapids.sh gs://$GCS_BUCKET/spark-gpu/rapids.sh
 popd
 pushd datasets/
 tar -xvf mortgage-small.tar.gz
-gsutil cp -r mortgage-small/ gs://$STORAGE_BUCKET/
+gsutil cp -r mortgage-small/ gs://$GCS_BUCKET/
 popd
 wget -O cudf-0.9.1-cuda10.jar https://search.maven.org/remotecontent?filepath=ai/rapids/cudf/0.9.1/cudf-0.9.1-cuda10.jar
 wget -O xgboost4j_2.11-1.0.0-Beta2.jar https://search.maven.org/remotecontent?filepath=ai/rapids/xgboost4j_2.11/1.0.0-Beta2/xgboost4j_2.11-1.0.0-Beta2.jar
 wget -O xgboost4j-spark_2.11-1.0.0-Beta2.jar https://search.maven.org/remotecontent?filepath=ai/rapids/xgboost4j-spark_2.11/1.0.0-Beta2/xgboost4j-spark_2.11-1.0.0-Beta2.jar
```

Comment:
Beta 3 jars

Reply:
done, also need to change maven url and from `xgboost4j_2.11` to `xgboost4j_2.x`